packages/fxa-content-server/app/tests/spec/lib/metrics.js

```diff
@@ -23,13 +24,24 @@ const MARKETING_CAMPAIGN = 'campaign1';
 const MARKETING_CAMPAIGN_URL = 'https://accounts.firefox.com';
 const BAD_METRIC_ERROR_PREFIX = 'Bad metric encountered:';
 
-describe('lib/metrics', () => {
+
+const performanceApis = [
+  { name: 'real - L2', api: getRealPerformanceApi(), useL1Timings: false },
+  { name: 'real - L1', api: getRealPerformanceApi(), useL1Timings: true },
+  { name: 'fallback', api: getFallbackPerformanceApi() }
+]
+
+for (const performanceApi of performanceApis) {
+
+
+describe('lib/metrics/' + performanceApi.name, () => {
```

Comment:
Indent level looks off here.

Reply:
Ah yes. I was on the fence about clobbering the history on the file or formatting. I'll format it though 👍 .